### PR TITLE
mTLS blog post

### PR DESCRIPTION
**Pull request by kflynn**
Signed-off-by: Flynn <flynn@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+- Without _authenticity_, an evildoer could easily pretend to be someone you
+  trust. mTLS provides authenticity using certificates.
+
+Note that we haven’t really said anything about authorization here. Much like
+with “normal” TLS, the only authorization that mTLS provides is the check that
+the certificates in play are valid and ultimately signed by a trusted issuer.
+Anything beyond that is up to something beyond mTLS: in our case, the service
+mesh.
+
+{{< cta-banner
```
Comment (CathPag) [inline linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md:174]: Flynn, can you remove the banner? We decided not to use them after all.

File: linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+  src="/uploads/2023/01/vecteezy-padlock-14719453-splash.png" >}}
+
+_This blog post is based on a workshop I recently delivered at Buoyant’s
+[Service Mesh Academy](https://buoyant.io/service-mesh-academy). If this seems
+interesting, check out the [full
+recording](https://buoyant.io/service-mesh-academy/kubernetes-mtls-with-linkerd)!_
+
+## mTLS and Linkerd
+
+You don’t have to spend much time in the cloud-native world before mTLS comes
```
Comment (CathPag) [inline linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md:25]:
```suggestion
You don’t have to spend much time in the cloud-native world before [mTLS](https://buoyant.io/mtls-guide) comes
```

File: linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+up. It shows up over and over again, especially once you start talking about
+zero trust. So what it is? What can it do, and what can it not do?
```
Comment (CathPag) [inline linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md:27]:
```suggestion
[zero trust](https://buoyant.io/resources/zero-trust-in-kubernetes-with-linkerd). So what it is? What can it do, and what can it not do?
```

File: linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+2. Workload B sends its certificate chain to the workload A, and workload A
+   checks to make sure that this chain is valid.
+
+3. Workload A then sends its certificate chain to workload B, and workload B
+   checks to make sure it’s valid.
+
+4. The two workloads start exchanging encrypted data.
+
+This is a small addition with a large impact. Authenticating both parties
+allows for meaningful authorization, which the service mesh can use to enforce
```
Comment (CathPag) [inline linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md:140]:
```suggestion
allows for meaningful authorization, which the [service mesh](https://buoyant.io/service-mesh-manifesto) can use to enforce
```

**Review by CathPag — COMMENTED** [review]
Added a few hyperlinks. We'll also need to remove the SMA banner.

File: linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md (hunk 1 above)
Comment (kflynn) [reply linkerd.io/content/blog/2023/0127-mtls-and-linkerd.md:174]: Done!

**Comment by kflynn** [comment]
Thanks, @CathPag! added those links.

**Review by jeremychase — APPROVED** [review]

**Pull request merged**